<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RssSettings</class>
 <widget class="QDialog" name="RssSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSS_FEED_SETTINGS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Resources/ltorrent.qrc">
    <normaloff>:/icons/app.ico</normaloff>:/icons/app.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="tbMenu">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../Resources/ltorrent.qrc">:/icons/app.ico</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LTitle">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>RSS_SETTINGS</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_centralWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>RSS_REFRASH_RATE</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="m_pRssUrlEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_pRefreshRateSpinBox">
        <property name="suffix">
         <string> MINUTES</string>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>RSS_URL</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>COOKIES</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QTableWidget" name="m_pCoociesTabWidget">
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="columnCount">
                <number>2</number>
               </property>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>19</number>
               </attribute>
               <row>
                <property name="text">
                 <string>Новая строка</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>COOKIE_ATTRIBUTE</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>COOLIE_VALUE</string>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="0" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>COOKIE_HINT</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QPushButton" name="addRowButton">
               <property name="text">
                <string>ADD_ROW</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="removeRowButton">
               <property name="text">
                <string>REMOVE_ROW</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pRssUrlEdit</tabstop>
  <tabstop>m_pRefreshRateSpinBox</tabstop>
  <tabstop>pbClose</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Resources/ltorrent.qrc"/>
  <include location="../../../Resources/ltorrent.qrc"/>
  <include location="../../../Resources/ltorrent.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RssSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RssSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addRowButton</sender>
   <signal>clicked()</signal>
   <receiver>RssSettings</receiver>
   <slot>onAddRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeRowButton</sender>
   <signal>clicked()</signal>
   <receiver>RssSettings</receiver>
   <slot>onRemoveRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddRow()</slot>
  <slot>onRemoveRow()</slot>
 </slots>
</ui>
